http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/DataprocRuntimeMonitor.scala

```diff
@@ -472,10 +472,6 @@ class DataprocRuntimeMonitor[F[_]: Parallel](
           _ <- setStagingBucketLifecycle(runtimeAndRuntimeConfig.runtime,
                                          config.runtimeBucketConfig.stagingBucketExpiration)
 
-          _ <- dbRef.inTransaction {
```

Comment:
this is when we delete a runtime. I don't think we need to update instances when we delete cluster. Hence deleting this